### PR TITLE
Add timeout arg

### PR DESCRIPTION
**Pull request by k-sever**
Make timeout configurable

### CODE REVIEW

**Comment by k-sever** [comment]
@lowobservable pls review

File: tn3270/telnet.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self, terminal_type, is_tn3270e_enabled=True):
         self.iac_buffer = bytearray()
         self.records = []
 
-    def open(self, host, port, ssl_context=None, ssl_server_hostname=None):
+    def open(self, host, port, ssl_context=None, ssl_server_hostname=None, timeout=None):
```
Comment (lowobservable) [inline tn3270/telnet.py:51]: Just a minor suggestion would be to reorder the parameters (keep the SSL ones at the end) and make it clearer that the timeout is just for the TN3270 negotiation and not the TCP/IP connection or something else...

```suggestion
    def open(self, host, port, tn3270_negotiation_timeout=None, ssl_context=None, ssl_server_hostname=None):
```

**Review by lowobservable — COMMENTED** [review]
Thanks again for contributing, sorry for the delay in getting back to you - just a minor suggestion about the parameter name and ordering on the `open()` method.

**Comment by lowobservable** [comment]
Thanks again @k-sever, I just published version `0.12.0` with your change.

**Pull request merged**